recipes/irma/meta.yaml

```diff
@@ -18,6 +18,9 @@ source:
 build:
   number: 0
   noarch: generic
+  run_exports:
+    weak:
```

Comment:
Could you please add a comment on why you chose this option?

Reply:
I have not finalized the PR yet, will update and ping you when I have it done. Thanks